pkg/test/framework2/common/settings.go

```diff
@@ -72,7 +71,7 @@ func (s *Settings) Clone() *Settings {
 // DefaultSettings returns a default settings instance.
 func DefaultSettings() *Settings {
 	return &Settings{
-		Environment: environment.DefaultName().String(),
+		Environment: DefaultEnvironmentName().String(),
```

Comment:
Why not make Settings.Environment typed ?

Reply:
Settings fields are used to collect from command-line as well.

I can take a stab at cleaning this up a bit more.